.changeset/fresh-boxes-relate.md

```diff
@@ -0,0 +1,7 @@
+---
+"@builder.io/packages": patch
+"@builder.io/sdks": patch
```

Comment:
this needs to be replaced with each SDK's name

Reply:
updated (in new file via CLI)